layers/+lang/plantuml/README.org

```diff
@@ -0,0 +1,28 @@
+#+TITLE: plantuml layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../css/readtheorg.css" />
+
+* Table of Contents                                        :TOC_4_org:noexport:
+ - [[Description][Description]]
+ - [[Install][Install]]
+ - [[Key bindings][Key bindings]]
+
+* Description
+This layer enables support for [[%5B%5Bfile:img/logo.png%5D%5D%0Ahttps://github.com/skuro/puml-mode][puml-mode]], which provides
+a major-mode for [[http://plantuml.com][plantuml]]. PlantUML is a tool to generate [[https://en.wikipedia.org/wiki/Unified_Modeling_Language][UML diagrams]] from plain-text.
+
+For help with how to use plantuml, see the [[http://plantuml.com][plantuml website]] and the [[http://plantuml.com/PlantUML_Language_Reference_Guide.pdf][reference guide]].
+
+
+* Install
+To use this contribution add it to your =~/.spacemacs=
+
+#+begin_src emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(plantuml))
+#+end_src
+
+* Key bindings
+
+| Key Binding              | Description                                           |
+|--------------------------+-------------------------------------------------------|
+| ~SPC m c c~ or ~C-c C-c~ | Build diagram from the text in the current buffer     |
+| ~SPC m o t~              | Set the output type (~unicode text~, ~svg~, or ~png~) |
```

Comment:
Not this one :-) https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#user-prefix